transcribed/page0310.txt

```diff
@@ -38,8 +38,8 @@
 038     001 006         MOVEM A,(J)
 039     001 016         POPJ P,
 040
-041     001 006 ASETMSK:        MOVE A,(J)     ;OPER 4
-042     001 025         TRZ A,BPIPI+VALRTF+BRKF+BCNTRZI ;MAKE SURE THESE GO TO SUP. PRO.
+041     001 006 ASETMSK:        MOVE A,(J)      ;OPER 4
+042     001 025         TRZ A,BPIPT+VALRTF+BRKF+BCNTRZI ;MAKE SURE THESE GO TO SUP. PRO.
```

Comment:
This is a tricky one, but we're now convinced it is BPIPI.

Reply:
Fixed